build-support/python/make-mesos-native-egg

```diff
@@ -139,38 +139,33 @@ CENTOS6_X86_64_DEPENDENCIES=(
   apr-devel
   cyrus-sasl-devel
   cyrus-sasl-md5
-  devtoolset-2-binutils
-  devtoolset-2-gcc
-  devtoolset-2-gcc-c++
+  devtoolset-7-binutils
+  devtoolset-7-gcc
+  devtoolset-7-gcc-c++
+  devtoolset-7-toolchain
   libcurl-devel
   make
   patch
   python27-python-devel
   subversion-devel
   zlib-devel
+  wget
+  tar
 )
 build_centos6() {
   local mesos_version=$1 output_basedir=$2
   local python_outdir=$output_basedir/centos/6/python
   local image_name=centos:6
 
   pushd "$TMPDIR"
-    docker run -i -v $(pwd):/mesos-egg $image_name bash <<EOF
+    docker run -m=2g -i -v $(pwd):/mesos-egg $image_name bash <<EOF
     set -e -u
 
     # We need Software Collections for Python 2.7 (we dropped support for 2.6).
     # wget must get installed before everything else to fetch the additional repos.
-    yum -y install centos-release-SCL wget
-
-    # Mesos requires gcc 4.8, which is available in devtools-2 on centos 6.
```

Comment:
Nit: We should leave a comment somewhere about why we're installing devtools-7 that way we know where to start looking if things break.